packages/rln/src/contract/constants.ts

```diff
@@ -1,7 +1,7 @@
 import { RLN_ABI } from "./abi.js";
 
 export const LINEA_CONTRACT = {
-  chainId: 59141,
+  chainId: "59141",
```

Comment:
is there any issues with consuming it in `ethers` or in a web app?

Reply:
I see that you do `parseInt` and `toString`
Let's not introduce repetitions. I'd better keep it as it is and convert where this value is written into `keystore` or read from `keystore`.